pkg/controllers/clusteroperator_controller.go

```diff
@@ -196,7 +196,9 @@ func (r *CloudOperatorReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Watches(&source.Kind{Type: &operatorv1.KubeControllerManager{}},
 			handler.EnqueueRequestsFromMapFunc(toClusterOperator),
 			builder.WithPredicates(kcmPredicates())).
-		Watches(&source.Channel{Source: watcher.EventStream()}, handler.EnqueueRequestsFromMapFunc(toClusterOperator))
+		Watches(&source.Channel{Source: watcher.EventStream()}, handler.EnqueueRequestsFromMapFunc(toClusterOperator)).
+		Watches(&source.Kind{Type: &corev1.ConfigMap{}}, handler.EnqueueRequestsFromMapFunc(toClusterOperator)).
+		Watches(&source.Kind{Type: &corev1.Secret{}}, handler.EnqueueRequestsFromMapFunc(toClusterOperator))
```

Comment:
So any configmap or secret change means we re-reconcile the cluster operator, which in turn leads us into the loop that manages the deployments?

Reply:
Yes. 

Actually, we do not care about a specific request content, reconcile loop is common.

Also, as far as I can say the only changes that matters are in `openshift-cloud-controller-manager` namespace. 
https://github.com/openshift/cluster-cloud-controller-manager-operator/blob/master/cmd/cluster-cloud-controller-manager-operator/main.go#L116